instances/subcellularEntity/nerveFiber.jsonld

```diff
@@ -0,0 +1,17 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/subcellularEntity/nerveFiber",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/SubcellularEntity",
+  "definition": "A threadlike extension of a nerve cell and consists of an axon and myelin sheath (if it is myelinated) in the nervous system.",
```

Comment:
Maybe better: 
"A threadlike extension of a nerve cell within the nervous system which consists of an axon and, if myelinated, a myelin sheath."